RELEASE-NOTES.txt

```diff
@@ -1,6 +1,5 @@
 16.6
 -----
-Reader post card and post details: added ability to mark a post as seen/unseen. [#15638, #15645]
```

Comment:
Why remove?

Reply:
As I stated in my commit note: "That was a bit premature as it's still feature flagged." I'll add it back when the feature is, ya know, enabled. 😄

Reply:
doH!  Would you believe I almost never read individual commit messages in PRs?  Now I have a reason to!

Reply:
Ha! Well, now I know where _not_ to put information you'd want to know. 😉